quic/s2n-quic/Cargo.toml

```diff
@@ -31,8 +31,9 @@ provider-tls-s2n = ["s2n-quic-tls"]
 
 # List of unstable features. Add new unstable features to the check in s2n-quic/src/lib.rs
 #
-# This depends on experimental behavior in s2n-tls.
+# These depend on experimental behavior in s2n-tls.
 unstable_client_hello = ["s2n-quic-tls/unstable_client_hello"]
+unstable_private_key = ["s2n-quic-tls/unstable_private_key"]
```

Comment:
I copied the client_hello callback and made the private key callback unstable, but I'm not sure it's necessary. Maybe we should consider our async callbacks stable now?

Reply:
I'd say let's let people try out the private key for now and if it works for them then let's stabilize.